extensions/blocks/subscriptions/attributes.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export const defaultAttributes = {
+	subscribePlaceholder: {
+		type: 'string',
+		default: __( 'Enter your email here', 'jetpack' ),
+	},
+	showSubscribersTotal: {
+		type: 'boolean',
+		default: false,
+	},
+	buttonOnNewLine: {
+		type: 'boolean',
+		default: false,
+	},
+	submitButtonText: {
+		type: 'string',
+		default: __( 'Sign Up', 'jetpack' ),
+	},
+	emailFieldBackgroundColor: {
+		type: 'string',
+	},
+	customEmailFieldBackgroundColor: {
+		type: 'string',
+	},
+	emailFieldGradient: {
+		type: 'string',
+	},
+	customEmailFieldGradient: {
+		type: 'string',
+	},
+	buttonBackgroundColor: {
+		type: 'string',
+		default: 'primary',
+	},
+	customButtonBackgroundColor: {
+		type: 'string',
+	},
+	buttonGradient: {
+		type: 'string',
+	},
+	customButtonGradient: {
+		type: 'string',
+	},
+	textColor: {
+		type: 'string',
+		default: 'background',
+	},
+	customTextColor: {
+		type: 'string',
+	},
+	fontSize: {
+		type: 'number',
+	},
+	customFontSize: {
+		type: 'number',
+		default: 18,
+	},
+	borderRadius: {
+		type: 'number',
+		default: 0,
+	},
+	borderWeight: {
+		type: 'number',
+		default: 1,
+	},
+	borderColor: {
+		type: 'string',
+	},
+	customBorderColor: {
+		type: 'string',
+	},
```

Comment:
I think the custom color values have to be hex, but the others can be names? Doing this means that the block inherits from the theme instead of forcing a color on users.